testcases/t/180-fd-leaks.t

```
@@ -20,43 +20,45 @@ use i3test;
 use POSIX qw(mkfifo);
 use File::Temp qw(:POSIX tempfile);
 
-my $i3 = i3(get_socket_path());
+if (-d '/proc/self') {
```

You are concerned the test will be silently skipped when you actually would like to be told something is wrong/changed with your procfs?
I see two options:
* skip with `if (qx/uname/ eq 'OpenBSD')`
* use `fstat` instead, which would be portable to OpenBSD